mp_api/client/core/client.py

```diff
@@ -179,9 +178,9 @@ def _create_session(api_key, include_user_agent, headers):
             mp_api_info = "mp-api/" + __version__ if __version__ else None
             python_info = f"Python/{sys.version.split()[0]}"
             platform_info = f"{platform.system()}/{platform.release()}"
-            session.headers[
-                "user-agent"
-            ] = f"{mp_api_info} ({python_info} {platform_info})"
+            session.headers["user-agent"] = (
+                f"{mp_api_info} ({python_info} {platform_info})"
+            )
```

Comment:
Is this an accidental change or different linter version? This is causing linting error.